tests/unit-tests/GravityView_Edit_Entry_Test.php

```
@@ -2763,6 +2763,148 @@ public function test_address_field_empty_string_fix() {
+						break;
+					}
+				}
+
+				return $has_logic;
+			},
+			999999,
+			2
+		);
+
+		// Use _emulate_render() to properly initialize Edit Entry context.
+		[ $output, $render, $entry ] = $this->_emulate_render( $form, $view, $entry );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$render'. (undefined)

(UnusedLocalVariable)